### ISSUE TITLE
@apply rule with brackets

### ISSUE THREAD

**Issue by JosefJezek** [pascalduez/postcss-apply#4]
Could you add support for `@apply` rule with brackets?

```css
    /* Shared Styles */
    --paper-font-common-base: {
      font-family: 'Roboto', 'Noto', sans-serif;
      -webkit-font-smoothing: antialiased;
    };
    --paper-font-common-nowrap: {
      white-space: nowrap;
      overflow: hidden;
      text-overflow: ellipsis;
    };
    /* Material Font Styles */
    --paper-font-display4: {
      @apply(--paper-font-common-base);
      @apply(--paper-font-common-nowrap);
      font-size: 112px;
      font-weight: 300;
      letter-spacing: -.044em;
      line-height: 120px;
    };
```

I need this brackets for Polymer `@apply` rule, check out this links
- https://github.com/PolymerElements/paper-styles/blob/master/typography.html
- https://www.polymer-project.org/1.0/docs/devguide/styling.html#custom-css-mixins

@robdodson Why Polymer have `@apply` rule with brackets? Proposal of `@apply` rule have not brackets, check out this link http://tabatkins.github.io/specs/css-apply-rule/

**Comment by pascalduez** [comment]
Same reply as in #3, the plugin scope is to follow the spec closely.

But since the spec is somewhat rough and very young, that seems like a reasonable addition.

**Comment by robdodson** [comment]
@JosefJezek hm good question (regarding the use of parens). @sorvell is there a reason why we need to use parens with `@apply`?

**Comment by JosefJezek** [comment]
@robdodson @sorvell Please any solution...

**Comment by pascalduez** [comment]
@JosefJezek I will add parens support, while implementing #3.  
Clearly missing time, but it's on reccords, no worry.

**Comment by JosefJezek** [comment]
@pascalduez Thank you :+1:

**Comment by pascalduez** [comment]
@JosefJezek Available as of `postcss-apply@0.2.0`
